### ISSUE TITLE
[Bug] Question about eta_min=0.8947368421052632(0.85 / 0.95) in CosineAnnealingMomentum config

### ISSUE THREAD

**Issue by ZhanghaiFA** [open-mmlab/mmdetection3d#3115]
### Prerequisite

- [x] I have searched [Issues](https://github.com/open-mmlab/mmdetection3d/issues) and [Discussions](https://github.com/open-mmlab/mmdetection3d/discussions) but cannot get the expected help.
- [x] I have read the [FAQ documentation](https://mmdetection3d.readthedocs.io/en/latest/notes/faq.html) but cannot get the expected help.
- [x] The bug has not been fixed in the [latest version (dev-1.x)](https://github.com/open-mmlab/mmdetection3d/tree/dev-1.x) or [latest version (dev-1.0)](https://github.com/open-mmlab/mmdetection3d/tree/dev-1.0).

### Task

I'm using the official example scripts/configs for the officially supported tasks/models/datasets.

### Branch

1.x branch https://github.com/open-mmlab/mmdetection3d/tree/dev-1.x

### Environment

Runtime environment:
    cudnn_benchmark: True
    mp_cfg: {'mp_start_method': 'fork', 'opencv_num_threads': 0}
    dist_cfg: {'backend': 'nccl'}
    seed: 895557828
    Distributed launcher: pytorch
    Distributed training: T

### Reproduces the problem - code sample

    dict(
        begin=0,
        by_epoch=True,
        convert_to_iter_based=True,
        end=2.4,
        eta_min=0.8947368421052632,  # 0.85/ 0.95
        type='CosineAnnealingMomentum'),
    dict(
        begin=2.4,
        by_epoch=True,
        convert_to_iter_based=True,
        end=6,
        eta_min=1,
        type='CosineAnnealingMomentum'),

### Reproduces the problem - command or script

Hi, quick question —

In the config, I saw:

eta_min = 0.8947368421052632  # 0.85 / 0.95

I understand that 0.85 and 0.95 are common momentum values, but since the official optimizer here is AdamW (with beta1 = 0.9), I’m not sure what this eta_min is doing.

Is it supposed to be a value between 0.85 and 0.95? Why is it written as a division?

Just trying to understand the reasoning behind this setting — any clarification would be appreciated!

Thanks!

### Reproduces the problem - error message

There is no error message — I’m just confused about the intention behind this line in the config:
